code/datums/materials/basemats.dm

```diff
@@ -246,3 +246,40 @@ Unless you know what you're doing, only use the first three numbers. They're in
 	. = ..()
 	if(istype(source, /obj/item))
 		qdel(source.GetComponent(/datum/component/fantasy))
+
+//Partical Smasher Strong
+
+/datum/material/valhollide
+	name = "valhollide"
+	id = "valhollide"
+	desc = "A powerful material made out of mashing particles into one another, ya science!"
+	color = "#6d7e8e"
+	strength_modifier = 1.1
+	categories = list(MAT_CATEGORY_RIGID = TRUE)
+	sheet_type = /obj/item/stack/sheet/mineral/valhollide
+	value_per_unit = 0.25 //Sure
+	beauty_modifier = 2 //It looks nice!
+	armor_modifiers = list("melee" = 1.8, "bullet" = 1.8, "laser" = 1.8, "energy" = 1.8, "bomb" = -1, "bio" = -1, "rad" = -1, "fire" = -1, "acid" = -4)
+
+/datum/material/morphium
+	name = "morphium"
+	id = "morphium"
+	desc = "A powerful material that mimics what the gray's use."
```

Comment:
```suggestion
	desc = "A powerful material that emulates the behaviors of advanced, rare alloys."
```

Reply:
I rather dislike that word choice but thanks